package.json

```diff
@@ -30,6 +30,17 @@
     "vite": "^2.9.6"
   },
   "dependencies": {
+    "@opentelemetry/api": "^1.1.0",
+    "@opentelemetry/auto-instrumentations-node": "^0.31.0",
+    "@opentelemetry/context-async-hooks": "^1.4.0",
+    "@opentelemetry/core": "^1.4.0",
+    "@opentelemetry/exporter-jaeger": "^1.3.1",
+    "@opentelemetry/instrumentation": "^0.30.0",
+    "@opentelemetry/propagator-ot-trace": "^0.26.0",
+    "@opentelemetry/resources": "^1.3.1",
+    "@opentelemetry/sdk-trace-base": "^1.3.1",
+    "@opentelemetry/semantic-conventions": "^1.3.1",
+    "dompurify": "^2.3.9",
```

Comment:
Same HTML sanitizer that Hydrogen uses, [ttps://github.com/vector-im/hydrogen-web/blob/.../src/platform/web/parsehtml.js#L66](https://github.com/vector-im/hydrogen-web/blob/c9bca52e82d96ae3545b3623c1a31522ddb30d91/src/platform/web/parsehtml.js#L66)

Link to project: https://github.com/cure53/DOMPurify